mppm/player-tags/target-instance.md

```diff
@@ -4,20 +4,20 @@ title: Use Player Tags to run a player as a server, client, or host
 description: This example shows how you can use Netcode for GameObjects to run a Player as a server, client, or host in Multiplayer Play Mode.
 ---
 
-This example explains how to target **Player Tags** in your game scripts with `CurrentPlayer.ReadOnlyTags()`. You can place these scripts where you want, but you must attach the scripts to a [NetworkObject](https://docs-multiplayer.unity3d.com/netcode/current/basics/networkobject/)(such as the **Player**).
+This example explains how to target **Player Tags** in your game scripts with `CurrentPlayer.ReadOnlyTags()`. You can place these scripts where you want, but you must attach the scripts to a [NetworkObject](https://docs-multiplayer.unity3d.com/netcode/current/basics/networkobject/) (such as the **Player**).
 
 You can also use the dedicated server package to set a player tag to server, client, or host. For more information, refer to ()[###LINK###].
```

Comment:
Please fix this. We should also add "Link" the list of terms we (or Vale) check for before merging.